Sources/SimpleSalesforceSDK/Models/SFAuth.swift

```diff
@@ -50,17 +50,14 @@ class SFAuth {
   func reset() {
     KeychainService.clearAll()
   }
-  
   /// Fetches a new access token only if it is already expired and stores it to Keychain
   /// - Parameters:
   ///     - config: The Salesforce instance’s configuration.
   /// This will also call `interospectAccessToken` to reset the expiry
   func refreshAccessTokenIfNeeded(config: SFConfig) async throws {
     guard !self.isAccessTokenValid else { return }
-    
-    try! await self.refreshAccessToken(config: config)
+    try await self.refreshAccessToken(config: config)
```

Comment:
how is this compiling? https://cocoacasts.com/what-is-the-difference-between-try-try-and-try

Reply:
nvm, I found the answer... looks like if the method has a `throws` in it, you can just use `try` 🎉

<img width="693" alt="Screen Shot 2022-08-05 at 3 54 31 PM" src="https://user-images.githubusercontent.com/528613/183151167-7dadeac5-a7b2-4186-80f9-14e031cd03c7.png">

https://stackoverflow.com/questions/32390611/try-try-try-what-s-the-difference-and-when-to-use-each